pom.xml

```diff
@@ -432,6 +394,15 @@
 				<scope>import</scope>
 			</dependency>
 
+			<!-- import BOM to lock all versions -->
+			<dependency>
+				<groupId>org.apache.logging.log4j</groupId>
+				<artifactId>log4j-bom</artifactId>
+				<version>${log4j2.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
```

Comment:
Can we now remove the 7 mentioned imports of this groupId as well? Or are they not included in this bom?

Reply:
Yes we can!